docs/tutorials/FlowTutorial.md

```diff
@@ -0,0 +1,1595 @@
+| `VERILOG_FILES`    | The path to the design Verilog files                                                                                                     |
+| `SDC_FILE`         | The path to design `.sdc` file                                                                                                           |
+| `CORE_UTILIZATION` | The core utilization percentage.                                                                                                         |
+| `PL_DENSITY`       | The desired placement density of cells. It reflects how spread the cells would be on the core area. 1 = closely dense. 0 = widely spread |
+
+### Timing Constraints
+
+View timing constraints specified in the `.sdc` file [here](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/master/flow/designs/sky130hd/ibex/constraint.sdc)
+
+```
+less ./designs/sky130hd/ibex/constraint.sdc
+```
+
+For `ibex` design we simply use the clock definition as follows as a
+minimum required timing constraint.
+
+```
+create_clock -name core_clock -period 17.4 [get_ports {clk_i}]
+```
+
+### Design Input Verilog
+
+The Verilog input files are located in `./designs/src/ibex/`
+
+The design is defined in `ibex_core.v` availabale [here](https://raw.githubusercontent.com/The-OpenROAD-Project/OpenROAD-flow-scripts/master/flow/designs/src/ibex/ibex_core.v)
+
+Refer to the file [ibex README.md](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/master/flow/designs/src/ibex/README.md)
+
+## Running The Automated RTL-to-GDS Flow
+
+This section describes the complete execution of the design flow from
+RTL-to-GDS. The OpenROAD application executes the complete, autonomous flow
+using Tcl scripts that invoke open sourced tools, from synthesis to the final
+`.gds` file creation, without requiring human intervention. In this tutorial,
+however, user will learn both the automated and a few interactive ways to
+run Tcl commands for important flow stages.
+
+From the OpenROAD-flow-scripts directory user can access individual flow
+stages, respective tools and the corresponding `README.md` for tool commands,
+configuration examples using the Tcl interface and other such details.
+
+-   Synthesis - [Yosys](https://github.com/The-OpenROAD-Project/yosys/blob/master/README.md)
+
+-   Floorplanning - [Initialize Floorplan](../../src/ifp/README.md)
+
+-   Global Placement - [RePlAce](../../src/gpl/README.md)
+
+-   Clock Tree Synthesis - [TrintonCTS 2.0](../../src/cts/README.md)
+
+-   Detailed Placement - [OpenDP](../../src/dpl/README.md)
+
+-   Global Route - [Fast Route](../../src/grt/README.md)
+
+-   Antenna Rule Checker - [Antenna Rule Checker](../../src/ant/README.md)
+
+-   Timing Optimization using the Resizer - [Gate Resizer](../../src/rsz/README.md)
+
+-   Detail Routing - [TritonRoute](../../src/drt/README.md)
+
+-   Layout Generation - [KLayout](https://www.klayout.de/) (Requires v0.27.1)

```

Comment:
remove empty lines

Reply:
@vijayank88 I meant that you had: `Synthesis-` instead of `Synthesis -` like the test of the table. No need to add lines. It looks like the missing space is resolved.

Reply:
He had asked for single whitespace between the words and the hyphen. Example: `Synthesis-` -> `Synthesis -`

The empty line between a list of items leads to the incorrect rendering of markdown and HTML.

Reply:
Thanks got it @vvbandeira  @arlpetergadfort